QgisModelBaker/libili2db/ili2dbconfig.py

```diff
@@ -200,16 +200,22 @@ def to_ili2db_args(self, extra_args=[], with_action=True):
 
         args += ["--coalesceCatalogueRef"]
         args += ["--createEnumTabs"]
-        args += ["--createNumChecks"]
+
+        if self.disable_validation:
+            args += ["--sqlEnableNull"]
+
+        if not self.disable_validation:
```

Comment:
```suggestion
        else:
```

Just a proof that I have actually done a review ;-)